src/app/[locale]/incident/add/components/questions/PlainText.tsx

```diff
@@ -0,0 +1,15 @@
+import { QuestionField } from '@/types/form'
+import Markdown from 'react-markdown'
+
+interface PlainTextProps extends Omit<QuestionField, 'register' | 'errors'> {}
+
+export const PlainText = ({ field }: PlainTextProps) => {
+  // TODO: Discuss if alert is the only used PlainText type in Signalen, style Markdown
+  return field.meta.value ? (
+    <div className="bg-red-100 rounded-lg p-4">
+      <Markdown>{field.meta.value}</Markdown>
```

Comment:
We kunnen misschien wel een herbruikbare versie maken van deze Markdown meets NL Design System component. Dan werkt de huisstijl ook door in deze content:

https://github.com/frameless/strapi/blob/main/packages/ui/src/components/Markdown/index.tsx

Misschien kunnen we wel een  `@utrecht/markdown-react` component maken.